projects/angular/src/utils/popover/popover-content.spec.ts

```diff
@@ -97,13 +97,14 @@ export default function (): void {
       it('responds to openChange events from the toggleService', function (this: Context) {
         this.testComponent.openState = true; // Add content to the DOM
         this.fixture.detectChanges();
-        const content = document.body.getElementsByClassName('clr-popover-content');
+        let content = document.body.querySelectorAll('div.clr-popover-content');
```

Comment:
A datepicker test (couldn't tell which one) left a `.clr-popover-content` zombie hanging out. I made this lookup specific to the test here, so it would no longer grab the zombie datepicker content.